src/assets/scripts/airport/Runway.js

```diff
@@ -97,7 +98,7 @@ export default class Runway {
             this.length = vlen(vsub(farSide.position, thisSide.position));
             // TODO: what is the 0.5 for? enumerate the magic number
             this.midfield = vscale(vadd(thisSide.position, farSide.position), 0.5);
-            this.angle = vradial(vsub(farSide.position, thisSide.position));
+            this.angle = radians_normalize(vradial(vsub(farSide.position, thisSide.position)));
```

Comment:
This should work for now, but I feel like this will need to be abstracted in `v3.0.x`.  What _exactly_ is this line doing; maybe something like `this.calculateRunwayAngle()`?

Doesn't have to be done now, but something to keep an eye out for.